docs/guide/security_checklist.md

```diff
@@ -115,9 +115,22 @@ Use the following checklist to ensure the Roles and Policies are secure:
 
 ## Underlying stack
 
+Once you have properly configured secure user roles and permissions, to avoid exposing your application to DDOS vulnerabilities and other, yet unknown security threats, make sure that you do the following:
 - Avoid exposing servers on the open internet when not strictly required.
 - Ensure any servers, services, ports and virtual hosts that were opened for testing purposes are locked down before going live.
 - Secure the database with a good password, keys, firewall, etc.
+- Consider whether you must make certain interfaces available on the open internet, for example:
```

Comment:
```suggestion
- Consider whether certain interfaces must be left available on the open internet. Roles protect your content on all interfaces, but you may prefer to reduce your attack surface. For example:
```
...because they are available by default. Also stressing that the services _are_ secure, this is an _optional extra_ level of security.